services/api-server/src/simcore_service_api_server/models/schemas/profiles.py

```diff
@@ -30,9 +30,30 @@ class Profile(ProfileCommon):
     groups: Optional[Groups] = None
     gravatar_id: Optional[str] = Field(
         None,
-        description="Hash value of email to retrieve an avatar image from https://www.gravatar.com",
+        description="md5 hash value of email to retrieve an avatar image from https://www.gravatar.com",
         max_length=40,
     )
 
     class Config:
-        schema_extra = {}
+        schema_extra = {
+            "example": {
+                "first_name": "James",
+                "last_name": "Maxwell",
```

Comment:
same here.

Reply:
These are all models and would like them this way so the python client uses snake case as well. 
I know the webserver API follows another notation but .. e.g. github API [does use snake-case](https://docs.github.com/en/rest/reference/projects)